ironfish/src/fileStores/config.ts

```diff
@@ -240,8 +240,66 @@ export type ConfigOptions = {
 }
 
 export const ConfigOptionsSchema: yup.ObjectSchema<Partial<ConfigOptions>> = yup
-  .object()
-  .shape({})
+  .object({
+    bootstrapNodes: yup.array().of(yup.string().defined()),
+    databaseName: yup.string(),
+    databaseMigrate: yup.boolean(),
+    editor: yup.string(),
+    enableListenP2P: yup.boolean(),
+    enableLogFile: yup.boolean(),
+    enableRpc: yup.boolean(),
+    enableRpcIpc: yup.boolean(),
+    enableRpcTcp: yup.boolean(),
+    enableRpcTls: yup.boolean(),
+    enableSyncing: yup.boolean(),
+    enableTelemetry: yup.boolean(),
+    enableMetrics: yup.boolean(),
+    getFundsApi: yup.string(),
+    ipcPath: yup.string(),
+    miningForce: yup.boolean(),
+    logPeerMessages: yup.boolean(),
+    // Already got a pareser to validate
+    logLevel: yup.string(),
+    logPrefix: yup.string().matches(RegExp('^([^%]*%(time|level|tag)%[^%]*)*$')),
```

Comment:
I think we should probably not require this regex here. If this regex gets out of sync with our logic to parse the logPrefix then it could prevent people from running their node. I think `yup.string()` is probably enough here

Reply:
I think the benefit justifies the use of a regex. Plus, it's only one place to change should the logic evolve in the future.